jsonpickle/pickler.py

```diff
@@ -780,17 +782,17 @@ def _flatten_dict_obj(self, obj, data=None):
         if self.keys:
             # Phase 1: serialize regular objects, ignore fancy keys.
             flatten = self._flatten_string_key_value_pair
-            for k, v in util.items(obj):
+            for k, v in util.items(obj, self.excludes):
```

Comment:
`for k, v in utils.items(obj, exclude=self.excludes):`  --  include the kwarg name at the call site here and below since it's optional.

Reply:
Agree.

Reply:
Agree